src/Microsoft.Extensions.Configuration.AzureAppConfiguration/ConfigurationClientManager.cs

```diff
@@ -334,5 +339,27 @@ private async Task RefreshFallbackClients(CancellationToken cancellationToken)
 
             _lastFallbackClientRefresh = DateTime.UtcNow;
         }
+
+        private async Task<bool> IsValidEndpoint(string hostName)
+        {
+            if (string.IsNullOrWhiteSpace(hostName))
+            {
+                return false;
+            }
+
+            if (string.IsNullOrEmpty(_validDomain))

```

Comment:
I was expecting valid domains to be an enumerable.

Reply:
updated